src/main/java/spaceinvaders/group_22/Player.java

```diff
@@ -144,5 +148,13 @@ public final void addLife() {
 	public final void setActivePowerUps(final ArrayList<PowerUp> newactivePowerUps) {
 		this.activePowerUps = newactivePowerUps;
 	}
+	/**
+	 * Let this player shoot a bullet.
+	 * @param bulletVelx speed of the bullet
+	 * @return Bullet that is shot.
+	 */
+	public final ShipBullet shootBullet(final double bulletVelx) {
```

Comment:
Why does a player shoot a bullet all of the sudden, this seems nog logical to me and conflicts with the responsibility of the player class. I think player.getSpaceShip().shootBullet() would be better.